src/main/java/seedu/address/model/tutorial/UniqueTutorialList.java

```diff
@@ -0,0 +1,150 @@
+package seedu.address.model.tutorial;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Iterator;
+import java.util.List;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import seedu.address.model.tutorial.exceptions.DuplicateTutorialException;
+import seedu.address.model.tutorial.exceptions.TutorialNotFoundException;
```

Comment:
Should there be a newline here after the imports

Reply:
![image](https://github.com/user-attachments/assets/d67f8962-142d-4e1c-9e2f-620b3b25b5b9)
The checkstyle is failing if I add a line separation, perhaps not?